ogticrd/covid-safe-paths
# Feat: qrcode generator

#### Description:


Adding QR code so users can share the app
#### Linked issues:


Feat: [284](https://trello.com/c/dqenkQFS/284-as-a-user-i-want-a-qr-code-so-that-i-can-share-the-app-with-friends)
#### Screenshots:


![image](https://user-images.githubusercontent.com/43874147/102545718-085fe200-408d-11eb-9b42-230346b43d99.png)


![image](https://user-images.githubusercontent.com/43874147/102544906-e2860d80-408b-11eb-8f21-580688edd1e3.png)

#### How to test:


- `git pull && git checkout Feat/284-qrcode-genertor`
-  `yarn install`
-  if you are using IOS after yarn install should do `npx pod-install` or `cd ios && pod install` 
-  Head over to Share section in settings 
-  Try to scan the QR code or share the app with other app like whatsapp and you should see the URL
